internal/faustjs.org/docs/getting-started.mdx

```diff
@@ -59,12 +59,12 @@ Once the necessary plugins have been installed, open the `.env.local` file you c
 NEXT_PUBLIC_WORDPRESS_URL=https://faustexample.wpengine.com
 
 # Plugin secret found in WordPress Settings->Headless
-# FAUSTWP_SECRET_KEY=YOUR_PLUGIN_SECRET
+FAUST_SECRET_KEY=YOUR_PLUGIN_SECRET
```

Comment:
I think you need to add a notice here for the version of Faust that this works correctly as this change is not backwards compatible. For example if I'm using Faust v 0.2.0 and with this merged PR the version of Faust becomes 0.3.0  then if I follow this reccomendation, my project will fail to build. Because we are supporting both FAUST_SECRET_KEY and FAUSTWP_SECRET_KEY we need to differentiate this in the docs.

Since we don't have Docs per Faust version this would be confusing.